src/libs/Navigation/types.ts

```diff
@@ -0,0 +1,401 @@
+/* eslint-disable @typescript-eslint/naming-convention  */
+import {CommonActions, NavigationContainerRefWithCurrent, NavigationHelpers, NavigationState, NavigatorScreenParams, PartialRoute, Route} from '@react-navigation/native';
```

Comment:
This file overall seems like a lot of duplication. Are you sure we can't derive these types, or is it better to define them separately in case of platform fragmentation?

Reply:
What part exactly? 
- Lines 36-399 is just definition for screens params, 
- Lines 12-34 is something that I had to extract from react navigation because of how we use navigation in the App. I don't think we can remove that part.
- Lines 8-10 is kind of redundant but helpful imo.